File: hulk/awnser.py
import sys


def build_hulk_feeling(layers_count: int) -> str:
    parts: list[str] = []

    for layer_index in range(1, layers_count + 1):
        if layer_index % 2 == 1:
            parts.append("I hate")
        else:
            parts.append("I love")

    return " that ".join(parts) + " it"


def main() -> None:
    data = sys.stdin.read().strip().split()
    if not data:
        return

    layers_count = int(data[0])
    result = build_hulk_feeling(layers_count)
    print(result)


if __name__ == "__main__":
    main()

File: hulk/awnser.go
package main

import (
	"bufio"
	"fmt"
	"os"
)

func buildHulkFeeling(layersCount int) string {
	parts := make([]string, 0, layersCount)

	for layerIndex := 1; layerIndex <= layersCount; layerIndex++ {
		if layerIndex%2 == 1 {
			parts = append(parts, "I hate")
		} else {
			parts = append(parts, "I love")
		}
	}

	result := ""
	for i, segment := range parts {
		if i > 0 {
			result += " that "
		}
		result += segment
	}
	result += " it"

	return result
}

func main() {
	in := bufio.NewReader(os.Stdin)
	out := bufio.NewWriter(os.Stdout)
	defer out.Flush()

	var layersCount int
	if _, err := fmt.Fscan(in, &layersCount); err != nil {
		return
	}

	fmt.Fprintln(out, buildHulkFeeling(layersCount))
}
